### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by issa-tseng**

### CODE REVIEW

**Review by matthew-white — APPROVED** [review]
🎉🎉🎉

File: lib/model/frames/submission.js [hunk 1]
```diff
@@ -7,43 +7,38 @@
...
-    const actorId = Option.of(maybeActor).map((actor) => actor.id).orNull();
-    const formattedDeviceId = blankStringToNull(deviceId);
-    return submissions.create(this, form, actorId, formattedDeviceId);
-  }
-
-  // just a stub for now; we don't need this feature just yet.
-  //createVersion(form, submission, maybeActor) { throw new Error('NYI'); }
+class Submission extends Frame.define(
+  table('submissions'),
+  'id',         readable,               'formId',       readable,
```
Comment (matthew-white) [inline lib/model/frames/submission.js:23]: I think we should consider not making these two fields readable: I think `instanceId` is the id that API users should use for the submission, not `id`; we also generally don't expose `formId` (as opposed to `xmlFormId`) over the API.

Comment (issa-tseng) [reply lib/model/frames/submission.js:23]: yeah that's oops

**Comment by issa-tseng** [comment]
this will be ready to merge as soon as slonik is merged upstream and updated here.

**Pull request merged**